alfresco-transform-tika/alfresco-transform-tika-boot/src/test/resources/quick.ott_metadata.json

```diff
@@ -1,6 +1,6 @@
-{
-  "{http://www.alfresco.org/model/content/1.0}description" : "Gym class featuring a brown fox and lazy dog",
-  "{http://www.alfresco.org/model/content/1.0}created" : 1126049640000,
-  "{http://www.alfresco.org/model/content/1.0}author" : "Jesper Steen Møller",
-  "{http://www.alfresco.org/model/content/1.0}title" : "The quick brown fox jumps over the lazy dog"
+{
+  "{http://www.alfresco.org/model/content/1.0}description" : "Gym class featuring a brown fox and lazy dog",
+  "{http://www.alfresco.org/model/content/1.0}created" : 1126060440000,

```

Comment:
Please revert this file. ott is not handled by IPTC extraction so should not change.